index.html

```diff
@@ -1790,15 +1790,25 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="caption">
 			<rdef>caption</rdef>
 			<div class="role-description">
-				<p>Visible content that names, and may also describe, a <rref>figure</rref>, <rref>table</rref>, <rref>grid</rref>, or <rref>treegrid</rref>.</p>
+				<p>Visible content that names, or describes a <rref>group</rref>, <rref>figure</rref>, <rref>table</rref>, <rref>grid</rref>, <rref>radiogroup</rref>, or <rref>treegrid</rref>.</p>
 				<p>When using <code>caption</code> authors SHOULD ensure:</p>
 				<ul>
-					<li>The <code>caption</code> is a direct child of a <rref>figure</rref>, <rref>table</rref>, <rref>grid</rref>, or <rref>treegrid</rref>.</li>
-					<li>The <code>caption</code> is the first child of a <rref>table</rref>, <rref>grid</rref>, or <rref>treegrid</rref>.</li>
-					<li>The <code>caption</code> is the first or last child of a <rref>figure</rref>.</li>
+					<li>The <code>caption</code> is a direct child of a <rref>group</rref>, <rref>figure</rref>, <rref>grid</rref>, <rref>radiogroup</rref>, <rref>table</rref>, or <rref>treegrid</rref>.</li>
```

Comment:
Maybe `direct child` be expanded to `owned child`. (I realize it's only a SHOULD statement but it seems like checkers shouldn't complain if it's owned.)

Reply:
```suggestion
					<li>The <code>caption</code> is an owned child of a <rref>group</rref>, <rref>figure</rref>, <rref>grid</rref>, <rref>radiogroup</rref>, <rref>table</rref>, or <rref>treegrid</rref>.</li>
```

Reply:
i had made the suggested edit, but deleted it as it is awkward per the following "caption is the first non-generic child of..." requirement.

to explain further, consider if I have the following markup

```
<div role=caption id=c ...>my amazing caption</div>
<div role=radiogroup aria-owns=c>
  <!-- all my radios go here -->
</div>
```
now the caption is 'owned' by the radiogroup.... but it gets slotted at the end of the radiogroup's content resulting as if the following markup were what had been written:

```
<div role=radiogroup ...>
  <!-- all my radios go here -->
  <div role=caption id=c>my amazing caption</div>
</div>
```

so now this fails the second requirement of the caption being the 'first child'.  

Someone would actually have to know to *not* have the radiogroup be the element to use aria-owns on to bring in the caption, but rather insert an empty div as the first child of the radiogroup and then have THAT element own the caption so that it could be slotted in at the right place.

```
<div role=caption id=c ...>my amazing caption</div>
<div role=radiogroup ...>
  <div aria-owns-c></div> <!-- insert an empty div to slot in the caption, rather than just putting the caption here. convoluted markup is convoluted -->
  <!-- all my radios go here -->
</div>
```

This again gets to my other comment that these requirements aren't actually useful in reality without name from encapsulation.

Reply:
Yikes - yes, that makes a lot of sense. (It's only a should in the end.)